flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/codegen/CollectorCodeGenerator.scala

```diff
@@ -73,19 +73,13 @@ class CollectorCodeGenerator(
     val input1TypeClass = boxedTypeTermForTypeInfo(input1)
     val input2TypeClass = boxedTypeTermForTypeInfo(collectedType)
 
-    // declaration in case of code splits
-    val recordMember = if (hasCodeSplits) {
-      s"private $input2TypeClass $input2Term;"
-    } else {
-      ""
-    }
+    val inputDecleration =
+      List(s"private $input1TypeClass $input1Term;",
+           s"private $input2TypeClass $input2Term;")
 
-    // assignment in case of code splits
-    val recordAssignment = if (hasCodeSplits) {
-      s"$input2Term" // use member
-    } else {
-      s"$input2TypeClass $input2Term" // local variable
-    }
+    val inputAssignment =
+      List(s"$input1Term = ($input1TypeClass) getInput();",
+           s"$input2Term = ($input2TypeClass) record;")
```

Comment:
Any particular reason you'd prefer to write it out directly inside the class code?  I was thinking grouping them into `inputAssignment` and `inputDeclaration` might make generated class code look more succinct, organized, and thus easier to read.

Reply:
Hi @haodang , I think variables should be used when they are variable. However, in the context, `inputAssignment` and `inputDeclaration` are constants.  So I think it would be more readable to put them in generated code, rather than guessing values of them.

Reply:
Okay, I'll write them explicitly in the generated class code.